search-relevancy-test/src/search_relevancy_test/anomaly_fetcher.clj

```diff
@@ -72,22 +74,24 @@
 (defn download-and-save-all-collections
   "Main function for this namespace to find all the concepts to download, retrieve them from
   the operational environment, and save them into the appropriate local testing directory."
-  []
-  (let [concept-ids (find-collection-ids-to-download)
-        concept-id->format-map (formats-for-collections concept-ids)]
-    (doseq [[concept-id fmt] concept-id->format-map]
-      (let [directory (get format-string->directory fmt)
-            filename (str concept-id (file-extension fmt))
-            full-path (format "%s/%s/%s" collections-dir directory filename)]
-        (if (.exists (io/as-file full-path))
-          (debug "Skipping already downloaded concept" concept-id)
-          (let [collection-metadata (download-concept-metadata concept-id fmt)]
-            (info "Saving" filename "to" directory)
-            (spit full-path collection-metadata)))))))
+  ([]
+   (download-and-save-all-collections core/provider-anomaly-filename))
+  ([filename]
+   (let [concept-ids (find-collection-ids-to-download filename)
+         concept-id->format-map (formats-for-collections concept-ids)]
+     (doseq [[concept-id fmt] concept-id->format-map]
+       (let [directory (get format-string->directory fmt)
+             filename (str concept-id (file-extension fmt))
+             full-path (format "%s/%s/%s" collections-dir directory filename)]
+         (if (.exists (io/as-file full-path))
+           (debug "Skipping already downloaded concept" concept-id)
+           (let [collection-metadata (download-concept-metadata concept-id fmt)]
+             (info "Saving" filename "to" directory)
+             (spit full-path collection-metadata))))))))
 
 (comment
- (download-and-save-all-collections)
- (find-collection-ids-to-download)
+ (download-and-save-all-collections core/edsc-anomaly-filename)
+ (find-collection-ids-to-download core/edsc-anomaly-filename)
  (formats-for-collections ["C1200196931-SCIOPS" "C1000000803-DEV08"]) ;; SIT
  (formats-for-collections ["C1397496671-NSIDC_ECS"]) ;; Prod
  (download-concept-metadata "C1237114193-GES_DISC"))
```

Comment:
Is this section here to run these tests manually?  Or is this just testing code that needs to be removed? Are we going to set up a periodic run?

Reply:
This is in a comment block for manual use.